code/style.css

```diff
@@ -1,5 +1,209 @@
-/* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
 
 body {
-  background-color: lightblue;
+  background-color: #F0F0F0;
 }
+
+h1 {
+  margin: 0;
+  text-align: center;
+  font-family: Helvetica;
+  font-weight: lighter;
```

Comment:
I see you have the `font-family` and `font-weight` the same in several places. If you want to use Helvetica in all or most of your website you can set the `font-family` and `font-weight` on the `body` tag. That way all of the content of the body will use that font and weight. Then if you have some alternate font to use in a specific place, like a `button` or some other headline, you can add the `font-family: { other font }` to that specific element.